universe/Effect.h

```diff
@@ -1123,7 +1123,7 @@ class FO_COMMON_API GenerateSitRepMessage : public EffectBase {
     const std::string& Icon() const
     { return m_icon; }
 
-    const auto& MessageParameters() const
+    auto MessageParameters() -> const std::vector<std::pair<std::string, ValueRef::ValueRefBase<std::string>*>>&
     { return m_message_parameters; }
```

Comment:
That is difficult and unfortunate.  I'd assumed that with a simple setting fix, you could save yourself the wait on the build bots.